### ISSUE TITLE
Unable to exclude `op_mini` from queries

### ISSUE THREAD

**Issue by ntwb** [ai/browserslist#65]
Running: `browserslist ">1%, not op_mini <= all"` results in `op_mini all` **not** being excluded:
```shell
and_chr 51
and_uc 9.9
android 4.4
chrome 51
chrome 50
chrome 49
chrome 29
edge 13
firefox 47
firefox 46
ie 11
ios_saf 9.3
ios_saf 9.0-9.2
op_mini all
safari 9.1
samsung 4
```

Testing other browsers works as expected:
• Running: `browserslist ">1%, not samsung <= 5"` correctly excludes `samsung 4`
• Running `browserslist ">1%, not chrome >= 50"` correctly excludes `chrome 51`, `chrome 50` but does include `chrome 49` and `chrome 29` which is correct.

I also tried `browserslist ">1%, not op_mini"` as the only value for `op_mini` is `all`

Have I missed something for non-numeric version numbers or is this a bug?

**Comment by ai** [comment]
There is no `all` version 😉.

**Comment by ntwb** [comment]
@ai In the code block above:

```
...
ios_saf 9.3
ios_saf 9.0-9.2
op_mini all           <----- How do I exclude this?
safari 9.1
samsung 4
...
```

**Comment by ai** [comment]
Seems like some new thing from Can I Use. Use `not op_mini all`.

**Comment by ntwb** [comment]
```shell
$ browserslist ">1%, not op_mini all"
browserslist: Unknown browser query `op_mini all`

**Comment by ai** [comment]
Ouh :(. Wait a second

**Comment by ai** [comment]
Fixed 36a93e04f0fff55ad9f1eca7a100dd5b0a24

**Comment by ai** [comment]
Released in 1.3.5

**Comment by ntwb** [comment]
Thanks 👍
